packages/database/src/cachers/EntityHierarchyCacher.js

```diff
@@ -88,11 +181,22 @@ export class EntityHierarchyCacher {
     return childIdToAncestorIds;
   }
 
-  async countEntityRelationChildren(hierarchyId, entityIds) {
-    return this.models.entityRelation.count({
-      parent_id: entityIds,
+  getEntityRelationChildrenCriteria(hierarchyId, parentIds) {
```

Comment:
This is just extending the pattern you suggested in a previous PR, to pull out the criteria to be more DRY (and change to use parentIds rather than entityIds for consistency)